gen/nvml/device.go

```diff
@@ -101,14 +101,14 @@ func (Device Device) GetSerial() (string, Return) {
 }
 
 // nvml.DeviceGetCpuAffinity()
-func DeviceGetCpuAffinity(Device Device, NumCPUs int) ([]uint, Return) {
+func DeviceGetCpuAffinity(Device Device, NumCPUs int) ([]uint32, Return) {
```

Comment:
The interface has remained as `unsigned long *` for the C definition, and this change from `uint` to `uint32` may be due to my platform. I will check up on how to better handle the `long` translation here tomorrow.

Reply:
This was due to the version of `c-for-go` that was update when building the Docker image I used to generate the bindings.